src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-subscriptions/lancie-admin-subscriptions.html

```diff
@@ -0,0 +1,121 @@
+<link rel="import" href="../../../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../../../bower_components/lancie-dialog/lancie-dialog.html">
+<link rel="import" href="../../../../bower_components/lancie-error/lancie-error.html">
+<link rel="import" href="../../../../bower_components/paper-spinner/paper-spinner-lite.html">
+
+<link rel="import" href="../../lancie-admin-page-layout.html">
+<link rel="import" href="lancie-admin-subscription.html">
+
+
+<dom-module id="lancie-admin-subscriptions">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+
+      [hidden] {
+        display: none !important;
+      }
+
+      paper-spinner-lite:not([active]) {
+        display: none;
+      }
+
+      .confirm-delete-button {
+        background-color: var(--paper-red-700);
+        color: #fff;
+      }
+    </style>
+
+    <lancie-ajax id="ajaxSubscriptions" auto-fire refurl="subscriptions" on-lancie-ajax="_onSubscriptionsResponse"></lancie-ajax>
+    <lancie-ajax id="ajaxDelete" refurl="subscriptions/[[deleting.id]]"
+                 on-lancie-ajax="_onDeleteSubscriptionResponse" method="DELETE"></lancie-ajax>
+
+    <lancie-admin-page-layout endpoint="subscriptions">
+      <p>This is a list of all the currently subscribed emails</p>
+      <lancie-error id="status"></lancie-error>
+      <paper-spinner-lite id="spinner" active="[[!subscriptions.length]]"></paper-spinner-lite>
+      <small id="empty" hidden>No subscriptions found!</small>
+      <template is="dom-repeat" items="[[subscriptions]]">
+        <lancie-admin-subscription item="[[item]]"></lancie-admin-subscription>
+      </template>
+    </lancie-admin-page-layout>
+
+    <lancie-dialog id="deleteSubscriptionDialog">
+      <h2>Confirm deletion</h2>
+      <div class="dialog-content">
+        <lancie-error id="dialogError"></lancie-error>
+        <h4>Are you certain you want to delete this subscription?</h4>
+        <p>[[deleting.email]]</p>
+      </div>
+      <div class="dialog-actions">
+        <paper-button dialog-dismiss>Cancel</paper-button>
+        <paper-button on-tap="handleDelete" class="confirm-delete-button">Delete</paper-button>
+      </div>
+    </lancie-dialog>
+
+  </template>
+
+  <script>
+    class LancieAdminSubscriptions extends Polymer.Element {
+      static get is() { return 'lancie-admin-subscriptions' }
+
+      static get properties() {
+        return {
+          subscriptions: {
+            type: Array,
+            value: []
+          }
+        }
+      }
+
+      constructor() {
+        super();
+        this.addEventListener('delete-subscription', this._onDeleteSubscription);
+      }
+
+      handleDelete() {
+        if (!isNaN(this.deleting.id)) {
+          this.subscriptions = [];
+          this.$.ajaxDelete.generateRequest();
+        }
+      }
+
+      _onSubscriptionsResponse(e, request) {
```

Comment:
This would be named `onDelete`. This way we have an ajax request `ajaxDelete` with the function that triggers it called `tryDelete` with the function on response called `onDelete`.